src/test/java/org/opensearch/ad/stats/ADStatsTests.java

```diff
@@ -71,16 +109,22 @@ public void setup() {
         nodeStatName2 = "nodeStat2";
 
         Settings settings = Settings.builder().put(MAX_MODEL_SIZE_PER_NODE.getKey(), 10).build();
-        SDKClusterService clusterService = mock(SDKClusterService.class);
-        SDKClusterSettings clusterSettings = clusterService.new SDKClusterSettings(
-            Settings.EMPTY, Collections.unmodifiableSet(new HashSet<>(Arrays.asList(MAX_MODEL_SIZE_PER_NODE)))
-        );
-        when(clusterService.getClusterSettings()).thenReturn(clusterSettings);
+        List<Setting<?>> settingsList = List.of(AnomalyDetectorSettings.MAX_MODEL_SIZE_PER_NODE);
+        ExtensionsRunner mockRunner = mock(ExtensionsRunner.class);
+        Extension mockExtension = mock(Extension.class);
+        when(mockRunner.getEnvironmentSettings()).thenReturn(settings);
+        when(mockRunner.getExtension()).thenReturn(mockExtension);
+        when(mockExtension.getSettings()).thenReturn(settingsList);
+        SDKClusterService sdkClusterService = new SDKClusterService(mockRunner);
+        // SDKClusterService.SDKClusterSettings clusterSettings = sdkClusterService.new SDKClusterSettings(
+        // Settings.EMPTY, Collections.unmodifiableSet(new HashSet<>(Arrays.asList(MAX_MODEL_SIZE_PER_NODE)))
+        // );
+        // when(sdkClusterService.getClusterSettings()).thenReturn(clusterSettings);
```

Comment:
Do we need this commented code here anymore?

Reply:
ACK